homeassistant/components/logbook/models.py

```diff
@@ -60,9 +57,6 @@ def __init__(
         self.event_type: str | None = self.row.event_type
         self.entity_id: str | None = self.row.entity_id
         self.state = self.row.state
-        self.context_id_bin: bytes | None = self.row.context_id_bin
-        self.context_user_id_bin: bytes | None = self.row.context_user_id_bin
-        self.context_parent_id_bin: bytes | None = self.row.context_parent_id_bin
```

Comment:
These are never accessed directly so setting them ahead of time defeated the purpose of having a lazy object